NeuroID/NeuroIDClass/Extensions/NIDAdvancedDevice.swift

```diff
@@ -92,4 +85,14 @@ public extension NeuroID {
             
         NeuroID.saveEventToLocalDataStore(nidEvent)
     }
+    
+    
+    @objc internal static func captureAdvancedDevice(_ shouldCapture: [Bool] = [NeuroID.isAdvancedDevice]) {
+        if shouldCapture[0] {
```

Comment:
I think this needs a backup in case for whatever reason the array doesn't have an item from the objc call